pipelines/release/run_rebuild.groovy

```diff
@@ -70,6 +70,11 @@ notify.wrap {
       ]
 
       def runJW = {
+        if (prepOnly) {
+          // need to ensure that no build/ststus.yaml exists
+          statusfile = "${cwd}/build/status.yaml"
+          boolean statusFileDeleted = new File(statusfile).delete()
```

Comment:
Let's try this, but there's a possibility that this needs to be a `sh "rm ${statusfile}"` on the worker node.  It's a little weird to have an unused variable.

Reply:
should we avoid to capture the result of the deletion in a variable?